eksctl/eksctl-cluster-config.yaml

```diff
@@ -0,0 +1,274 @@
+# This eksctl configuration file represents the cluster and node groups for use
+# by the cluster.
+# ref: https://eksctl.io/usage/schema/
+#
+# Cluster operations:
+# ref: https://eksctl.io/usage/cluster-upgrade/
+#
+#   create:   eksctl create cluster --config-file=eksctl-cluster-config.yaml --set-kubeconfig-context
+#   upgrade:  eksctl upgrade cluster --config-file=eksctl-cluster-config.yaml
+#   delete:   eksctl delete cluster --config-file=eksctl-cluster-config.yaml
+#
+# Node group operations:
+# ref: https://eksctl.io/usage/managing-nodegroups/
+#
+#   eksctl get nodegroups --cluster jmte
+#
+#   eksctl delete nodegroup --config-file=eksctl-cluster-config.yaml --include "user-a-*,worker-a-*" --approve
+#   eksctl create nodegroup --config-file=eksctl-cluster-config.yaml --include "user-a-*,worker-a-*"
+#   eksctl delete nodegroup --cluster jmte --name core-a
+#   eksctl create nodegroup --cluster jmte --name core-a
+#
+#   eksctl delete nodegroup --config-file=eksctl-cluster-config.yaml --include "user-a-*,worker-a-*" --approve && eksctl create nodegroup --config-file=eksctl-cluster-config.yaml --include "user-a-*,worker-a-*"
+#
+# Attribution: this was based on @yuvipanda's work in 2i2c! <3
+# ref: https://github.com/2i2c-org/pangeo-hubs/blob/8e552bc198d8339efe8c003cb847849255e8f8ed/aws/eksctl-config.yaml
+#
+
+
+
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: jmte
+  # region:
+  #   The region was chosen to to us-west-2 (Oregon) to be close to a CMIP-6
+  #   dataset.
+  #
+  region: us-west-2
+  version: "1.19"
```

Comment:
I think 1.20 is available from AWS, are you selecting 1.19 to be in sync with the rest of the pilots-hubs stuff, I guess, just curious :wink: ?

Reply:
Sorry for super late response @damianavila! I'm not sure what I thought any more at that time, but I guess I'd cared to avoid piloting usage of the k8s manifests on a newer k8s version than they have previously been used on.